kalamine/utils.py

```diff
@@ -35,8 +43,30 @@ class Layer(IntEnum):
     ALTGR = 4
     ALTGR_SHIFT = 5
 
+    def next(self) -> 'Layer':
+        """The next layer in the layer ordering."""
+        return Layer(int(self)+1)
+
+    def necromance(self) -> 'Layer':
+        """Remove the effect of the dead key if any."""
+        if self == Layer.ODK:
+            return Layer.BASE
+        elif self == Layer.ODK_SHIFT:
+            return Layer.SHIFT
+        return self
```

Comment:
Nice. The name is totally debatable but I love this idea.

Reply:
Yes it was a bit late when I designed this joke. Feel free to revert of course. The thing that I think really matters is to have a clear function with clear meaning instead of a random `-2`.

Reply:
Hahaha I feel so stupid. Thanks for the good catch. :-)